scripts/workflow-state.sh

```
@@ -199,36 +282,94 @@ cmd_summary() {
         echo ""
     fi
 
+    # Debug investigation findings
+    if [ "$workflow_type" = "debug" ]; then
+        local findings=$(jq -r '.investigation.findings[]? // empty' "$state_file" | head -5)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)